contracts/0.8.9/EIP712StETH.sol

```diff
@@ -53,9 +53,9 @@ contract EIP712StETH is IEIP712StETH {
 
         bytes32 hashedName = keccak256(bytes("Liquid staked Ether 2.0"));
         bytes32 hashedVersion = keccak256(bytes("2"));
-        bytes32 typeHash = keccak256(
+        bytes32 typeHash = keccak256(bytes(
```

Comment:
I didn't get why we needed it, tbh :thinking: 

OZ doesn't do that (both for our outdated 4.4 version and the up-to-date 4.8.x)

Reply:
only for uniformity (above we bring the string to bytes, but not here)

i think we can remove this commit

interesting  case:
internal comment, with bytes keccak256(bytes("test")) ~ 419gas, just string keccak256("test") ~ 337 gas.
- because with bytes it push text to stack and use sha3 op for encode
- for pure keccak256 it calculate hash immediately, i see only push32 {hash} op

Reply:
Yeah, it makes sense for consistency. 

Though, I would rather remove casting to `bytes` for the cases above. WDYT?

Reply:
done